test/async_cache_test.dart

```diff
@@ -13,17 +13,16 @@ void main() {
 
   setUp(() {
     // Create a cache that is fresh for an hour starting at 1/1/17 @ 12:30PM.
-    cache = new AsyncCache<String>(const Duration(hours: 1), now: () => time);
     time = new DateTime(2017, 1, 1, 12, 30);
+    cache = new AsyncCache<String>(const Duration(hours: 1), now: () => time);
   });
 
   test('should fetch via a callback when no cache exists', () async {
-    var callback = expectAsync0/*<Future<String>>*/(() async => 'Expensive');
-    expect(await cache.fetch(callback), 'Expensive');
+    expect(await cache.fetch(() async => 'Expensive'), 'Expensive');
   });
 
   test('should not fetch via callback when a cache exists', () async {
-    Future<String> doNotCall() async => 'Should not have been invoked...';
+    var doNotCall = expectAsync0/*<Future<String>>*/((){}, count: 0);
```

Comment:
Nit: include these inline in the `cache.fetch()` calls. That'll avoid the need to name them, and the need to explicitly annotate.